### ISSUE TITLE
[Enhancement] Font: It may be better to use "SF Pro" or "Inter" before "NotoSans SC"

### ISSUE THREAD

**Issue by bamboo512** [Yidadaa/ChatGPT-Next-Web#501]
On iOS/iPadOS and macOS, the default font for the user interface is **SF Pro** for Latin characters and **PingFang SC** for Simplified Chinese characters. Using these fonts can add a unique and attractive aesthetic to your user interface.

On Windows, although **SF Pro** isn't pre-installed, you can install SF Pro manually to let the browser use **SF Pro**, or use Inter, which closely resembles **SF Pro**.

If using Inter, you can even use **Google Fonts** to load network font resources.

Here are the `font-family` that I recommend: 

```
body {
    font-family: ui-sans-serif, "SF Pro", "Inter", "Noto Sans SC", "PingFang SC", "SF Pro Icons", "Arial", sans-serif;
}

.markdown-body code, .markdown-body pre {
	font-family: ui-monospace, "SF Mono", "JetBrains Mono", "IBM Plex Mono", "Fira Code", "Roboto Mono", "Consolas", "Monaco", "Andale Mono", "Ubuntu Mono", monospace;
}
```

> **Note**
>
> `ui-sans-serif` and `ui-monospace` can only be used in Safari.

**Comment by bamboo512** [comment]
In this screenshot, I am using **SF Pro** and **SF Mono** as the UI font.
![](https://user-images.githubusercontent.com/16702370/230079844-0033b6e5-3144-47cc-ac72-125df58bd622.png)

**Comment by Yidadaa** [comment]
Noto Sans Font is loaded from google fronts service now.

**Comment by bamboo512** [comment]
> Noto Sans Font is loaded from google fonts service, so you think it is better to use SF Font as default UI font on Mac OS?
Yes, it may look better if using **SF Pro** and **SF Mono** on Apple platforms. Additionally, since [Inter](https://fonts.google.com/specimen/Inter) can also be loaded from Google Fonts, I highly recommend you to use Inter on other platforms.

**Comment by Yidadaa** [comment]
We have already using Noto Sans font on every platforms, which is loaded from Google Fonts. So a better solution is to change the order of css font-family, place SF font before Noto Sans?
